src/components/pages/TeamPage.jsx

```diff
@@ -20,66 +21,23 @@ const TeamPage = () => {
 
         <h2 className="font-spartan font-semibold text-2xl lg:text-3xl"> Team Mentors</h2>
         <div className="w-full flex flex-wrap items-center justify-center mb-4">
-          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} imgShift="12" textShift="8"/>
-          <MentorCard BlobImage={YellowBlobImage} teamMemberData={teamMembers[1]} cardShift="2"/>
-          <MentorCard BlobImage={PurpleBlobImage} teamMemberData={teamMembers[2]} cardShift="6"/>
+          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} cardShift="0" imgShift="24" textShift="8" />
+          <MentorCard BlobImage={YellowBlobImage} teamMemberData={teamMembers[1]} cardShift="2" />
+          <MentorCard BlobImage={PurpleBlobImage} teamMemberData={teamMembers[2]} cardShift="6" />
         </div>
 
         <h2 className="font-spartan font-semibold text-2xl lg:text-3xl mb-6">Team Members</h2>
         <div className="w-full flex flex-row flex-wrap items-center justify-center gap-10 mb-10">
-
-          <div className="w-60 basis-60 flex flex-col items-center gap-3">
-            <img
-              src={teamMembers[3].src}
-              alt={teamMembers[3].alt}
-              className="w-36 h-36 rounded-full border-solid border-4 border-darkGreen"
-            />
-            <p className="w-full font-spartan font-semibold text-xl lg:text-2xl text-center">{teamMembers[3].name}</p>
-            <p className="w-full font-spartan font-semibold text-lg lg:text-xl text-center">{teamMembers[3].title}</p>
-          </div>
-
-          <div className="w-60 basis-60 flex flex-col items-center gap-3">
-            <img
-              src={teamMembers[4].src}
-              alt={teamMembers[4].alt}
-              className="w-36 h-36 rounded-full border-solid border-4 border-darkGreen"
-            />
-            <p className="w-full font-spartan font-semibold text-xl lg:text-2xl text-center">{teamMembers[4].name}</p>
-            <p className="w-full font-spartan font-semibold text-lg lg:text-xl text-center">{teamMembers[4].title}</p>
-          </div>
+          {[...Array(teamMembers.length).keys()]
```

Comment:
You can just iterate over the array with a map or filter function. No need for this. `[...Array(teamMembers.length).keys()]`

Reply:
Got it. Thank you!  
I removed `[...Array(teamMembers.length).keys()]`